tools/opt/opt.cpp

```diff
@@ -112,28 +126,119 @@ NOTE: The optimizer is a work in progress.
                Join two blocks into a single block if the second has the
                first as its only predecessor. Performed only on entry point
                call tree functions.
-  -h, --help   
+  -O
```

Comment:
I would not advertise -O and -Os until we are fairly confident they are close to what we want. We want to make sure initial experience with these options is positive. If you follow my recommendations for what to include for them, then I would be fine with advertising here.

Reply:
I think marking it [experimental] is the way to go.  No point hiding it if we want users to try it out.